docs/source/tutorials/e2e_flow.rst

```diff
@@ -164,78 +227,68 @@ modifying its associated config ``eleuther_evaluation.yaml``.
     to install the EleutherAI evaluation harness.
 
 Since we plan to update all of the checkpoint files to point to our fine-tuned checkpoints,
-let's first copy over the config to our local working directory so we can make changes. This
-will be easier than overriding all of these elements through the CLI.
+let's first copy over the config to our local working directory so we can make changes.
 
 .. code-block:: bash
 
     tune cp eleuther_evaluation ./custom_eval_config.yaml \
 
-For this tutorial we'll use the `truthfulqa_mc2 <https://github.com/sylinrl/TruthfulQA>`_ task from the harness.
-This task measures a model's propensity to be truthful when answering questions and
-measures the model's zero-shot accuracy on a question followed by one or more true
-responses and one or more false responses. Let's first run a baseline without fine-tuning.
```

Comment:
Any particular reason for deleting this? Seems to me it'd still be relevant to know how to run the baseline

Reply:
I would have to rerun the full fine tuning. It didn’t seem essential.  Can we have it as a follow up 😂?

Reply:
I did run the eval script though, and it works